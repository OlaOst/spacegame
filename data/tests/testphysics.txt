test.position = [0.0, 0.0]
test.velocity = [0.0, 0.0]
test.force = [0.0, 1.0]
test.mass = 1.0