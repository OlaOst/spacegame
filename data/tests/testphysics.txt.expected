test.position = [0, 0]
test.velocity = [0, 1]
test.force = [0, 1]